doc/tutorial/index.rst

```diff
@@ -206,8 +238,22 @@ just the opcode:
     };
 
 .. spicy-output:: tftp-1.spicy
-    :exec: cat tutorial/examples/udp-contents.orig.1367411051.972852.dat | spicy-driver %INPUT
-    :show-as: cat udp-contents.orig.1367411051.972852.dat | spicy-driver tftp.spicy
+    :exec: spicy-driver -F tutorial/examples/tftp_rrq.dat -P 69/udp=TFTP::Packet -P 50618/udp=TFTP::Packet %INPUT
+    :show-as: spicy-driver -F tftp_rrq.dat -P 69/udp=TFTP::Packet -P 50618/udp=TFTP::Packet tftp.spicy
+    :max-lines: 6
+
+As you see, we now use ``-P 69/udp=TFTP::Packet`` because we no longer
+need to worry about the direction: from now on, the same ``Packet``
+unit handles both originator and responder sides. However, because the
+way TFTP works, we need an additional parser mapping for the data
+connection that's part of the PCAP as well, because that happens on a
+different port: ``-P 50618/udp=TFTP::Packet``. (The handling of such
+dynamic, non-standard ports is something that normally the host
+application---e.g., Zeek---would handle on its side. With
+``spicy-driver``, we need to do it manually ourselves.)
```

Comment:
Could the part in parentheses be a note? It feels weird to basically have a whole couple of sentences in parentheses